src/CinemaList/SingleCinema.js

```diff
@@ -4,24 +4,71 @@ import ButtonBack from './ButtonBack';
 import './SingleCinema.css';
 
 class SingleCinema extends Component {
+  state = {
+    showTechnology: false,
+    showDescrioption: false
+  }
+
+  handleClickTechnology = ()=>{
+    const show = this.state.showTechnology;
+    this.setState({ showTechnology: !show });
+  }
+
+  handleDescriptionClick =()=> {
+    const isShow = this.state.showDescrioption;
+    this.setState({ showDescrioption: !isShow });
+  }
+
   cinema =() => {
     const target = CinemaStore[this.props.match.params.id-1];
     return (
       <div>
         <div className="cinema-img-box">
-        <img src={target.url} className="cinema-img" alt="cinema"/>
+        <img src={target.url} className="cinema-img" alt={target.name}/>
         </div>
         <h1 className="cinemaholl-name">{target.name}</h1>
         <p className="cinemaholl-street">{target.street}</p>
-        <h1 className="seans">Розклад сеансів</h1>
       </div>
     )
   }
+
   render() {
-    return(
-      <div className="cinema2" onClick={this.cinema}>
 
+    let technology = null;
+    if(this.state.showTechnology) {
+      const target = CinemaStore[this.props.match.params.id - 1];
+      technology = (
+        <div className="technology-box">{target.technology.split('\n').map((item, key) => {
+          return <p key={key}>{item}<br /></p>
+        })}</div>
+      )
+    }
+
+    let description = null;
+
+    if(this.state.showDescrioption) {
+      const target = CinemaStore[this.props.match.params.id - 1];
+
+      description = (
+        <div className="description-box">{target.description.split('\n').map((item, key) => {
+          return <p key={key}>{item}<br /></p>
+        })}</div>
+      )
+    }
+    return(
+      <div className="cinema-holl" onClick={this.cinema}>
```

Comment:
You don't need to use so complicated logic inside the render method. Please move it into separate methods and will be better to divide logical block's onto different components.